include/fmt/base.h

```diff
@@ -2811,13 +2813,39 @@ inline auto runtime(string_view s) -> runtime_format_string<> { return {{s}}; }
 
 /** Formats a string and writes the output to ``out``. */
 template <typename OutputIt,
-          FMT_ENABLE_IF(detail::is_output_iterator<OutputIt, char>::value)>
-auto vformat_to(OutputIt out, string_view fmt, format_args args) -> OutputIt {
+          FMT_ENABLE_IF(detail::is_output_iterator<remove_cvref_t<OutputIt>,
+                                                   char>::value)>
+auto vformat_to(OutputIt&& out, string_view fmt, format_args args)
+    -> remove_cvref_t<OutputIt> {
   auto&& buf = detail::get_buffer<char>(out);
   detail::vformat_to(buf, fmt, args, {});
   return detail::get_iterator(buf, out);
 }
 
+template <typename OutputIt, typename OutputSen = OutputIt>
+struct format_to_result {
+  /** Iterator past the end of the last write. */
```

Comment:
It is not super clear what "last write" means. In `format_to_n_result` we have

```c++
/** Iterator past the end of the output range. */
```

but I guess it can now be ambiguous since output range can denote either the range of the output produced by a function or a passed output "range". Any ideas how to rephrase it better?

Reply:
I tried a different formulation to make it a bit more clear: 

```cpp
/** Iterator pointing to just after the last succesful write in the range. */
```